internal/node/test/bootstrap_with_patch.js

```diff
@@ -0,0 +1,7 @@
+// bootstrap the bazel require patch since this bootstrap script is loaded with
+// `--node_options=--require=$(rlocation $(location :bootstrap_with_patch.js))`
+if (process.env['BAZEL_NODE_RUNFILES_HELPER']) {
+  require(process.env['BAZEL_NODE_RUNFILES_HELPER']).patchRequire();
+}
+console.log('here')
```

Comment:
meant to remove this?

Reply:
just part of a test log but may as well go 👍